changelogs/unreleased/gh-5928-ability-to-pass-several-uris-with-properties-in-different-ways.md

```diff
@@ -0,0 +1,5 @@
+## feature/core
+
+* Implemented ability to pass several listening URIs in different ways
+  with different properties. URIs can be passed either as a string or
+  as a table, or as a combination of these options.
```

Comment:
Please remove this changelog entry as well as the docbot request. The fact that one can now pass query parameters to `box.cfg.listen` makes no sense to our users until we have any parameters that can be used with it.

Reply:
fixed